### DISCUSSION TITLE
Landscape layout of display

### DISCUSSION THREAD

**Discussion by blauwkeut** [Jyers/Marlin#74] [Ideas]
Not sure if it's even remotely possible, but an open question to see whether there would be a way to use the display in a landscape orientation?

Reason: I'm using Octoprint with a PiTFT50, so would be cool if I could use a housing that holds both screens on top of each other.
If not, no worries at all, just wondering...

**Comment by Schnabulation** [comment]
...there is a guy on reddit who rewrote the DWIN code for landscape mode, but I think it's equally difficult than the rewrite from Jyers. I doubt that he is going to implement it.
